EXAMPLES.md

```diff
@@ -45,6 +45,21 @@ ShopifyBuy.Init(accessToken, shopDomain);
 
 After you initialize the SDK, you can use `ShopifyBuy.Client()` to query Shopify. You need to initialize the SDK only once.
 
+## Supporting multiple languages
+
+If your store supports multiple languages, then the Storefront API can return translated content for supported resource types and fields.
```

Comment:
Maybe tweak the second half to: `, then you can use the Storefront API to return translated content for supported resource types and fields.` just to match the subjects.